### DISCUSSION TITLE
Chart creation

### DISCUSSION THREAD

**Discussion by dineshshekar** [ShapeCrawler/ShapeCrawler#528] [Q&A]
Is it possible to create charts? i can see that we can update the existing chart values but not able to  add the categories and points dynamically.

**Comment by AdamShakhabov** [comment]
No, chart creation, adding a new chart category, and adding chart data point are not yet available. But in process🙂

**Reply by zzzaaammm** [reply]
Is this still in process or paused?

**Reply by AdamShakhabov** [reply]
It is still in pause status🙄 and waiting for a volunteers.

**Reply by chenyu-vmware** [reply]
> It is still in pause status🙄 and waiting for a volunteers.

I also have such requirement, does chart creation, adding a new chart category, and adding chart data point are supported now? thanks.

**Reply by AdamShakhabov** [reply]
The creation of some chart types is already possible. Please check this doc https://github.com/ShapeCrawler/ShapeCrawler/blob/master/examples/Charts.cs
